dropwizard-bom/pom.xml

```diff
@@ -24,8 +24,9 @@
         <dropwizard.version>${project.version}</dropwizard.version>
         <guava.version>19.0</guava.version>
         <jersey.version>2.22.2</jersey.version>
-        <jackson.api.version>2.6.0</jackson.api.version>
-        <jackson.version>2.6.5</jackson.version>
+        <jackson.api.version>2.7.3</jackson.api.version>
+        <jackson.version>2.7.3</jackson.version>
+        <jackson.jaxrs.version>2.7.2</jackson.jaxrs.version>
```

Comment:
@arteam Will try to get the rest (XML and Avro modules, JAX-RS) out today. Just wanted to verify one or two things on XML module. So yes, would be great to wait just one more day if possible.